src/client/theme-default/components/NavBarLinks.vue

```diff
@@ -7,6 +7,7 @@
       </template>
     </template>
     <NavBarLink v-if="repoInfo" :item="repoInfo" />
+    <NavDropdownLink v-if="localeCandidates" :item="localeCandidates" />
```

Comment:
Lets move this one above the GitHub repo nav 🙏

Reply:
Moved!